README.md

```diff
@@ -1,2 +1,191 @@
+### Enable API's
+In order to operate with the Service Account you must activate the following APIs on the project where the Service Account was created:
+
+- Compute Engine API - compute.googleapis.com
+- Kubernetes Engine API - container.googleapis.com
+
+## Install
+
+### Terraform
+Be sure you have the correct Terraform version (0.10.x), you can choose the binary here:
+- https://releases.hashicorp.com/terraform/
+
+## Usage
+There are multiple examples included in the [examples](./examples/) folder but simple usage is as follows:
+
+```hcl
+module "gke" {
+  source                     = "github.com/terraform-google-modules/terraform-google-kubernetes-engine"
+  project_id                 = "<PROJECT ID>"
+  cluster_name               = "gke-test-1"
+  region                     = "us-central1"
+  network                    = "vpc-01"
+  subnetwork                 = "us-central1-01"
+  ip_range_pods              = "us-central1-01-gke-01-pods"
+  ip_range_services          = "us-central1-01-gke-01-services"
+  node_service_account       = "project-service-account@<PROJECT ID>.iam.gserviceaccount.com"
+  http_load_balancing        = false
+  horizontal_pod_autoscaling = true
+  kubernetes_dashboard       = true
+  network_policy             = true
+
+  node_pools = [
+    {
+      name         = "default-node-pool"
+      machine_type = "n1-standard-2"
+      min_count    = 1
+      max_count    = 100
+      disk_size_gb = 100
+      disk_type    = "pd-standard"
+      image_type   = "COS"
+      auto_repair  = true
+      auto_upgrade = true
+    },
+  ]
+
+  node_pools_labels = {
+    all = {}
+
+    default-node-pool = {
+      default-node-pool = "true"
+    }
+  }
+
+  node_pools_taints = {
+    all = []
+
+    default-node-pool = [
+      {
+        key    = "default-node-pool"
+        value  = "true"
+        effect = "PREFER_NO_SCHEDULE"
+      },
+    ]
+  }
+
+  node_pools_tags = {
+    all = []
+
+    default-node-pool = [
+      "default-node-pool",
+    ]
+  }
+}
+```
+
+Then perform the following commands on the root folder:
+
+- `terraform init` to get the plugins
+- `terraform plan` to see the infrastructure plan
+- `terraform apply` to apply the infrastructure build
+- `terraform destroy` to destroy the built infrastructure
+
+#### Variables
+Please refer the /variables.tf file for the required and optional variables.
+
+#### Outputs
+Please refer the /outputs.tf file for the outputs that you can get with the `terraform output` command
+
+## Infrastructure
+The resources/services/activations/deletions that this module will create/trigger are:
+- Create a GKE cluster with the provided addons
+- Create GKE Node Pool(s) with provided configuration and attach to cluster
+- Replace the default kube-dns configmap if `stub_domains` are provided
+- Activate network policy if `network_policy` is true
+- Add `ip-masq-agent` configmap with provided `masq_non_masquerade_cidrs` if `network_policy` is true or `masq_config_enabled` is true
```

Comment:
Change `masq_non_masquerade_cidrs` to `non_masquerade_cidrs`.

Reply:
Done.